quesma/testdata/aggregation_requests.go

```diff
@@ -5371,35 +5371,39 @@ var AggregationTests = []AggregationTestCase{
 				"total": 1
 			},
 			"aggregations": {
-				"2": {
+				"0": {
 					"buckets": [
 						{
-							"doc_count": 1,
-							"key": 9100.0
-						},
-						{
-							"doc_count": 0,
-							"key": 9200.0
-						},
-						{
-							"doc_count": 0,
-							"key": 9300.0
-						},
-						{
-							"doc_count": 0,
-							"key": 9400.0
-						},
-						{
-							"doc_count": 0,
-							"key": 9500.0
```

Comment:
Don't worry, this test was unfinished and had complete rubbish here, now it's actually a fully proper test.